parsing/dialects/dialect1/parserImpls.ftl

```diff
@@ -517,10 +517,12 @@ SqlTableAttribute TableAttributeJournalTable() :
 SqlTableAttribute TableAttributeMap() :
 {
     final SqlIdentifier id;
+    SqlIdentifier colocateName = null;
 }
 {
     <MAP> <EQ> id = CompoundIdentifier()
-    { return new SqlTableAttributeMap(id, getPos()); }
+    [ <COLOCATE> <USING> colocateName = SimpleIdentifier() ]
```

Comment:
Is it ever legal for the colocateName to be a CompoundIdentifier?

Reply:
It looks like it is valid, it's changed to CompoundIdentifier()